content/operate/rc/supported-regions.md

```diff
@@ -0,0 +1,140 @@
+| `europe-north1`     | Finland             | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `europe-southwest1` | Madrid, Spain       | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `europe-west1`      | Belgium             | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `europe-west2`      | London, UK          | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `europe-west3`      | Frankfurt, Germany  | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `europe-west4`      | Netherlands         | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `europe-west6`      | Zurich, Switzerland | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+### Asia Pacific
+
+| Region ID              | Location             | Redis Cloud Pro | Redis Cloud Essentials |
+|------------------------|----------------------|-----------------|------------------------|
+| `asia-east1`           | Taiwan               | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-east2`           | Hong Kong, China     | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-northeast1`      | Tokyo, Japan         | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `asia-northeast2`      | Osaka, Japan         | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-northeast3`      | Seoul, South Korea   | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-south1`          | Mumbai, India        | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `asia-south2`          | Delhi, India         | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-southeast1`      | Singapore            | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-southeast2`      | Jakarta, Indonesia   | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `australia-southeast1` | Sydney, Australia    | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `australia-southeast2` | Melbourne, Australia | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+### Middle East
+
+| Region ID  | Location         | Redis Cloud Pro | Redis Cloud Essentials |
+|------------|------------------|-----------------|------------------------|
+| `me-west1` | Tel Aviv, Israel | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+## Microsoft Azure
```

Comment:
Can we put more clear dividers (maybe also bigger font) between the cloud providers? It looks pretty similar to the dividers between the different locations insider each provider.

Reply:
I made the location headings smaller, let me know what you think.